packages/samples/react/src/scenarios/static-form.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+	KolButton,
+	KolInputCheckbox,
+	KolInputColor,
+	KolInputDate,
+	KolInputEmail,
+	KolInputFile,
+	KolInputNumber,
+	KolInputPassword,
+	KolInputRadio,
+	KolInputRange,
+	KolInputText,
+	KolSelect,
+	KolTextarea,
+} from '@public-ui/react';
+import type { FC } from 'react';
+import React from 'react';
+import { SampleDescription } from '../components/SampleDescription';
+
+export const StaticForm: FC = () => {
+	return (
+		<>
+			<SampleDescription>
+				<p>This sample shows how you can use KoliBri in static form context, too.</p>
+				<ol>
+					<li>
+						First you have to enable the <code>experimental mode</code>:{' '}
+						<code className="bg-gray-200">&lt;meta name=&quot;kolibri&quot; content=&quot;dev-mode=false;experimental-mode=true;&quot; /&gt;</code>
+					</li>
+					<li>
+						Actually you have to use a native <code>form</code> element:{' '}
+						<code className="bg-gray-200">&lt;form method=&quot;GET&quot;&gt;...&lt;/form&gt;</code>
+					</li>
+					<li>
+						It is important in static usage to give every input a <code>name</code> attribute, because the form data will be sent as a query string:{' '}
```

Comment:
Den Teilsatz würde ich vielleicht weglassen: `, because the form data will be sent as a query string:`

Das Formular kann ja auch als FormData verschickt werden, das würde ich aber nicht alles beschrieben.